lib/routes/informedainews/docs.ts

```diff
@@ -0,0 +1,78 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { load } from 'cheerio'; // 类似 jQuery 的 API HTML 解析器
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/zh-Hans/docs/:type',
+    categories: ['journal'],
+    example: '/docs/world-news-daily',
+    parameters: { type: 'world-news-daily|tech-enthusiast-daily|ai-enthusiast-daily' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['informedainews.com', 'informedainews.com/zh-Hans/docs/:type', 'informedainews.com/docs/:type'],
+            target: '/zh-Hans/docs/:type',
+        },
+    ],
+    name: '知闻AI',
+    maintainers: ['xirizhi'],
```

Comment:
```suggestion
    maintainers: ['guicaiyue'],
```
This one should be your GitHub ID.